ui/app/components/app/asset-list-item/asset-list-item.js

```diff
@@ -40,6 +58,36 @@ const AssetListItem = ({
     )
     : null
 
+  const sendTokenButton = useMemo(() => {
```

Comment:
Choosing to memoize this here because it is unlikely to change for any given tokenSymbol/address combo, and because it prevents the Button from rendering on every render of this component. It achieves the latter by also making it so that the onClick handler isn't recreated every render.